### PR TITLE
Feature/animation fixes

### PR DESCRIPTION
**Pull request by martinfalk**
* some fixes for the animation module

### CODE REVIEW

**Review by ResearchDaniel — DISMISSED** [review]
lgtm

File: modules/animation/src/animationcontroller.cpp [hunk 1]
```diff
@@ -129,6 +129,9 @@ AnimationController::AnimationController(Animation& animation, InviwoApplication
     playOptions.setCollapsed(true);
     addProperty(playOptions);
 
+    framesPerSecond.onChange([this]() { setPlaySpeed(framesPerSecond); });
```
Comment (petersteneteg) [inline modules/animation/src/animationcontroller.cpp]: Should not setPlaySpeed update frames per second too? Don't know why there is two members with same state...

**Review by petersteneteg — CHANGES_REQUESTED** [review]

File: modules/animation/src/animationcontroller.cpp (hunk 1 above)
Comment (martinfalk) [reply modules/animation/src/animationcontroller.cpp]: `deltaTime_` is the inverse of framesPerSecond and also holds the playback direction (signed delta).

Comment (martinfalk) [reply modules/animation/src/animationcontroller.cpp]: going for OrdinalRefProperty to hold the state only once.

File: modules/animation/src/animationcontroller.cpp [hunk 2]
```diff
@@ -65,15 +65,20 @@ size_t imageExtIndex(InviwoApplication* app, std::string_view ext) {
...
                      0)
     , playWindow("PlayFirstLastTime", "Window", 0, 10, 0, 1e5, 1, 0.0,
                  InvalidationLevel::InvalidOutput, PropertySemantics::Text)
-    , framesPerSecond("PlayFramesPerSecond", "Frames per Second", 25.0, 000.1, 1000.0, 1.0,
-                      InvalidationLevel::InvalidOutput, PropertySemantics::Text)
+    , framesPerSecond(
+          "PlayFramesPerSecond", "Frames per Second",
+          [this]() { return 1.0 / std::abs(deltaTime_.count()); },
+          [this](const double& value) { deltaTime_ = Seconds(1.0 / value); },
+          {000.1, ConstraintBehavior::Immutable}, {1000.0, ConstraintBehavior::Immutable}, 1.0,
```
Comment (ResearchDaniel) [inline modules/animation/src/animationcontroller.cpp]: 000.1 -> 0.001

**Review by petersteneteg — DISMISSED** [review]

**Review by ResearchDaniel — APPROVED** [review]

**Review by petersteneteg — APPROVED** [review]

**Pull request merged**